app/src/main/java/org/mozilla/focus/tabs/TabViewClient.java

```diff
@@ -0,0 +1,32 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.tabs;
+
+/**
+ * An abstract layer of @see{android.webkit.WebViewClient}
+ */
+public class TabViewClient {
```

Comment:
I'm not sure if we'd like to move the non-WebviewClient implementations to another class/Wrapper to be compatible for future WebViewClient changes?